ansible_base/jwt_consumer/common/auth.py

```diff
@@ -143,9 +146,13 @@ def parse_jwt_token(self, request):
 
         logger.info(f"User {self.user.username} authenticated from JWT auth")
 
-    def log_and_raise(self, conditional_translate_object, expand_values={}):
+    def log_and_raise(self, conditional_translate_object, expand_values={}, error_code=None):
         logger.error(conditional_translate_object.not_translated() % expand_values)
-        raise AuthenticationFailed(conditional_translate_object.translated() % expand_values)
+        translated_error_message = conditional_translate_object.translated() % expand_values
+        if error_code == 498:
+            raise InvalidTokenException(translated_error_message)
```

Comment:
Just to confirm, you've tested and this exception type gets properly picked up in the login flow? From a quick glance it does not look like anything is requiring AuthenticationFaield exception above this call.

Reply:
Here are results from a live server testing with these changes:
```
Feb 25 15:56:15 ip-10-0-0-54 automation-eda-controller[92302]: 2025-02-25 15:56:15,591 ansible_base.jwt_consumer.common.auth ERROR    JWT expired 788 seconds ago - check for clock skew. Request ID: a7b69775-f179-41ad-8cd6-0ede0d2bafe5
```
<img width="1128" alt="Screenshot 2025-02-25 at 10 58 26 AM" src="https://github.com/user-attachments/assets/ed7df462-e733-4c86-b8f3-cbd7ef8ed1a4" />

<img width="1128" alt="Screenshot 2025-02-25 at 10 58 39 AM" src="https://github.com/user-attachments/assets/bfe0a810-c4e6-438d-b154-da7cb5158478" />